src/portfft/committed_descriptor_impl.hpp

```diff
@@ -148,18 +148,30 @@ class committed_descriptor_impl {
     std::vector<kernel_data_struct> transpose_kernels;
     std::shared_ptr<IdxGlobal> factors_and_scan;
     detail::level level;
+    // The size of DFT transform which will be computed for the given dimension
     std::size_t length;
+    // The committed length for the particular dimension, will be different from length in the case of bluestein and
+    // radar fft algorithms
```

Comment:
Yes the length is the one which will be padded in case of Bluestein. The comment above  `length` 
>     // The size of DFT transform which will be computed for the given dimension

Indicating the size of the transform which will be computed in the kernel. I have avoided the word padding on purpose here, because in rader (thanks for catching the spelling mistake), we do not pad, we subtract, and hence I have kept the description generic without mentioning any padding. The `length` can be greater than (Bluestein), Less Than (Rader) or equal to ( cooley tukey) to the committed size

Reply:
I didn't realise Rader would be less. 

This is clear-ish, but I think its worth repeating yourself here to really hammer home the difference, given how important these two variables are. Maybe describe the committed length additionally as *user-supplied*, and the other as *internal*.

Reply:
> I didn't realise Rader would be less.

Yes, Rader converts an N point DFT to a N - 1 Cyclic convolution, which in-turn is calculated using FFT's, very useful for sizes like 65537.

Sure I can add that phrase

Reply:
Done